### PR TITLE
New version: Tencent.QQMusic version 18.51

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41735)

### CODE REVIEW

File: manifests/t/Tencent/QQMusic/18.51/Tencent.QQMusic.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Tencent.QQMusic
+PackageVersion: "18.51"
+DefaultLocale: en-US
```
Comment (Trenly) [inline manifests/t/Tencent/QQMusic/18.51/Tencent.QQMusic.yaml:6]:
```suggestion
DefaultLocale: zh-CN
```

File: manifests/t/Tencent/QQMusic/18.51/Tencent.QQMusic.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- aac
+- mp3
+- 腾讯
+- Tencent
+# Agreements: 
+ReleaseNotes: |
+  - 「音质」海量 Hi-Res 音质歌曲支持在线播放和下载啦
+  - 「其他」问题修复及用户体验优化
+# ReleaseNotesUrl: 
+ManifestType: locale
```
Comment (Trenly) [inline manifests/t/Tencent/QQMusic/18.51/Tencent.QQMusic.locale.zh-CN.yaml:38]:
```suggestion
ManifestType: defaultLocale
```

File: manifests/t/Tencent/QQMusic/18.51/Tencent.QQMusic.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- artist
+- playlist
+- aac
+- mp3
+- Tencent
+- QQ音乐
+# Agreements: 
+# ReleaseNotes: 
+# ReleaseNotesUrl: 
+ManifestType: defaultLocale
```
Comment (Trenly) [inline manifests/t/Tencent/QQMusic/18.51/Tencent.QQMusic.locale.en-US.yaml:36]:
```suggestion
ManifestType: locale
```

**Review by OfficialEsco — APPROVED** [review]
I'm pretty sure en-US should always be the defaultLocale even tho the package is fully Chinese

**Comment by Trenly** [comment]
> I'm pretty sure en-US should always be the defaultLocale even tho the package is fully Chinese

Then whats the point of the defautLocale?

**Comment by OfficialEsco** [comment]
> > I'm pretty sure en-US should always be the defaultLocale even tho the package is fully Chinese
> 
> Then whats the point of the defautLocale?

The default Metadata to Display, if the user have a different preferred locale it will show the locale for that language.

**Pull request merged**